components/main.css

```diff
@@ -0,0 +1,13 @@
+main {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.image {
```

Comment:
generally it's better to use sizes relative to the containing elements (% values). if you use values relative to the viewport, things can break easily when resizing the window. In this specific case you used `vh` for both wight and width - I suppose you meant to use `vw` for the width...

Reply:
I fixed it. There are some other issues I will work on over the week-end.